src/compiler/config/outputs/validate-dist.ts

```diff
@@ -41,7 +44,7 @@ export const validateDist = (config: d.ValidatedConfig, userOutputs: d.OutputTar
       systemDir: config.buildEs5 ? lazyDir : undefined,
       systemLoaderFile: config.buildEs5 ? join(lazyDir, namespace + '.js') : undefined,
       legacyLoaderFile: join(distOutputTarget.buildDir, namespace + '.js'),
-      polyfills: distOutputTarget.polyfills !== undefined ? !!distOutputTarget.polyfills : true,
+      polyfills: true,
```

Comment:
Since we're actually ensuring (below) that `.polyfills` is always a boolean here this like ends up being equivalent to just always `true`...is this what we what? should the `polyfill` property on `DIST_LAZY` instead always be what the value on `DIST` is? that's what I think would make the most sense to me, but the tests as-written seem to require that it's always `true`

Reply:
Wait, shouldn't this be `distOutputTarget.polyfills`?

Reply:
I'll go ahead and change it to that though since it really does seem like The Right Thing